helx/charts/roger/templates/roger-data-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: roger-data-pvc
```

Comment:
Would be good to base the name off of the chart release name, that way multiple deployments can be done in the same namespace.

Reply:
This is true I agree but code that uses this (https://github.com/helxplatform/roger/blob/02bd4cb4edacc165520675d3e0f6f6da67023eca/dag_util.py#L76) needs to be updated aswell, maybe we need to track it as an issue(?)

Reply:
This isn't a huge deal, but it's easier to deal with now since it is fresh in your mind.  There are similar issues in other charts (like the pod names).  Trying to figure out how to get the naming right later can be troublesome.  Maybe change it up when you get a chance in a future commit/merge.

Reply:
Sounds good will do in the next Roger develop branch update and here.